packages/core/src/provisioning.ts

```diff
@@ -351,4 +378,154 @@ export class BasicProvisioner implements Provisioner {
+
+        if (!accountId || !orgId) {
+            httpRes.statusCode = 400;
+            httpRes.end("Invalid SCIM Secret Token");
+            return;
+        }
+
+        try {
+            const body = await readBody(httpReq);
+            newUser = JSON.parse(body);
+        } catch (e) {
+            httpRes.statusCode = 400;
+            httpRes.end("Failed to read request body.");
+            return;
+        }
+
+        const validationError = this.validateScimUser(newUser);
+        if (validationError) {
+            httpRes.statusCode = 400;
+            httpRes.end(validationError);
+            return;
+        }
+
+        try {
+            const provisioning = await this.getProvisioning({ email: newUser.email, accountId });
+
+            const organization = provisioning.orgs.find((org) => org.id === orgId);
+
+            if (!organization) {
+                throw new Error("Organization not found");
+            }
+
+            // TODO: create user
+            // const invite = new Invite(email, purpose);
+            // await invite.initialize(org, this.account!);
+            // await org.addOrUpdateMember(invite.invitee);
+            console.log(JSON.stringify({ provisioning, organization }, null, 2));
+        } catch (error) {
+            console.error(error);
+            httpRes.statusCode = 500;
+            httpRes.end("Unexpected Error");
+            return;
+        }
+
+        httpRes.statusCode = 200;
+        httpRes.end();
+    }
+
+    handleScimPost(httpReq: IncomingMessage, httpRes: ServerResponse) {
+        const url = new URL(`http://localhost${httpReq.url || ""}`);
+        switch (url.pathname) {
+            // TODO: Implement this
+            // case "/Groups":
+            //     return this.handleScimGroupsPost(httpReq, httpRes);
+            case "/Users":
+                return this.handleScimUsersPost(httpReq, httpRes);
+            default:
+                httpRes.statusCode = 404;
+                httpRes.end();
+        }
+    }
+
+    async handleScimRequest(httpReq: IncomingMessage, httpRes: ServerResponse) {
+        switch (httpReq.method) {
+            case "POST":
+                return this.handleScimPost(httpReq, httpRes);
+            // TODO: Implement these
+            // case "PATCH":
+            //     return this.handleScimPatch(httpReq, httpRes);
+            // case "DELETE":
+            //     return this.handleScimDelete(httpReq, httpRes);
+            default:
+                httpRes.statusCode = 405;
+                httpRes.end();
+        }
+    }
+
+    async startScimServer() {
+        // TODO: Remove this
+        console.log("======== provisioning.startScimServer");
+        const server = createServer((req, res) => this.handleScimRequest(req, res));
+        server.listen(this.config!.port);
+    }
+}
+
+// TODO: Maybe it makes sense to move this from the http server transport to core's transport as readHttpBody?
```

Comment:
As discussed, simply move the class/module into the `@padloc/server` package and then you can simply import the function from `@padloc/server/src/transport/http`.